src/components/Markdown/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  Anchor,
+  Box,
+  Image as GrommetImage,
+  Markdown as GrommetMarkdown,
+  Paragraph,
+} from 'grommet';
+import { Download, Github } from 'grommet-icons';
+import PropTypes from 'prop-types';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { dark as codestyle } from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+class Image extends React.Component {
+  render() {
+    const { src } = this.props;
+    if (src === 'Github') {
+      return <Github color="brand" />;
+    }
+    if (src === 'Download') {
+      return <Download color="brand" />;
+    }
+    return <GrommetImage {...this.props} />;
+  }
+}
+
+Image.propTypes = {
+  src: PropTypes.string,
+};
+

```

Comment:
Let's document this Image component with comments, it's not quite clear how this is implemented in the markdown.

Reply:
I was thinking about actually making an Anchor that smartly adds the right icon depending on the URL so this way of using grommet icons via markdown isn't needed

Reply:
Will do in the next PR